src/App.tsx

```diff
@@ -1,204 +1,67 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos, USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import { Header } from './components/Header';
+import { Footer } from './components/Footer';
+import { TodoList } from './components/TodoList';
+import { ErrorMessage } from './components/ErrorMessage';
+import { Errors } from './types/Errors';
+import { Status } from './types/Status';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+  const [status, setStatus] = useState<string>(Status.All);
+
+  useEffect(() => {
+    getTodos()
+      .then(data => setTodos(data))
+      .catch(() => {
+        setErrorMessage(Errors.UnableToLoad);
+        const timer = setTimeout(() => setErrorMessage(''), 3000);
+
+        return () => clearTimeout(timer);
```

Comment:
This `return` should be from `useEffect`, not from `catch`(it should work as a `ComponentWillUnmount`)